examples/carousel/carousel-2-tablist.html

```diff
@@ -0,0 +1,797 @@
+              </tr>
+            </thead>
+            <tbody>
+              <tr data-test-id="rotation-aria-label">
+                <td></td>
+                <th scope="row">
+                  <code>aria-label</code>
+                </th>
+                <td>
+                  <code>button</code>
+                </td>
+                <td>
+                  The accessible name is updated when the rotation is paused or palying to reflect the current action the button will perform:
+                  <ul>
+                    <li>Paused: "Start automatic slide show".</li>
+                    <li>Playing: "Stop automatic slide show".</li>
+                  </ul>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </section>
+        <section>
+          <h3 id="rps_label_2">Tabs</h3>
+          <table aria-labelledby="rps_label_2 rps_label" class="data attributes">
+            <thead>
+              <tr>
+                <th scope="col">Role</th>
+                <th scope="col">Attribute</th>
+                <th scope="col">Element</th>
+                <th scope="col">Usage</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <th scope="row">
+                  <code>region</code>
+                </th>
+                <td></td>
+                <td>
+                  <code>section</code>
+                </td>
+                <td>
+                  <ul>
+                    <li>Role <code>region</code> is implied for any <code>section</code> element that has an accessible name.</li>
+                    <li>Defines the carousel and its controls as a land mark region.</li>
+                  </ul>
+                </td>
+              </tr>
+              <tr data-test-id="carousel-region-aria-roledescription">
+                <td></td>
+                <th scope="row">
+                  <code>aria-roledescription="carousel"</code>
+                </th>
+                <td>
+                  <code>section</code>
+                </td>
+                <td>
+                  <ul>
+                    <li>Informs assistive technologies to identify the element as a &quot;carousel&quot; rather than a &quot;region.&quot;</li>
+                    <li>Effects how the assistive technology renders the role but does not effect functionality, such as commands for navigating to landmark regions.</li>
+                  </ul>
+                </td>
+              </tr>
+              <tr data-test-id="carousel-region-aria-label">
+                <td></td>
+                <th scope="row">
+                  <code>aria-label="Highlighted television shows"</code>
+                </th>
+                <td>
+                  <code>section</code>
+                </td>
+                <td>Provides a label that describes the content in the carousel region.</td>
+              </tr>
+              <tr data-test-id="carousel-aria-live">
```

Comment:
test needed for this data-test-id

Reply:
@spectranaut added test for data-test-id="carousel-aria-live"